src/pomodoro/containers/PomodoroControls.js

```diff
@@ -74,40 +74,68 @@ class PomodoroControl extends React.PureComponent {
   }
 }
 
-const PomodoroControls = ({ status, play, pause, stop, skip }) => (
-  <Container>
-    <PomodoroControl
-      icon='stop'
-      size='small'
-      design='tertiary'
-      onClick={stop}
-      disabled={Boolean(status === 'IDLE')}
-    />
-
-    {status === 'RUNNING' ? (
-      <PomodoroControl
-        icon='pause'
-        onClick={pause}
-        width='28'
-        height='28'
-      />
-    ) : (
-      <PomodoroControl
-        icon='play'
-        onClick={play}
-        width='38'
-        height='38'
-      />
-    )}
-
-    <PomodoroControl
-      icon='skip'
-      size='small'
-      design='tertiary'
-      onClick={skip}
-    />
-  </Container>
-)
+class PomodoroControls extends React.PureComponent {
```

Comment:
I think we can add all this keyboard logic to another component, maybe something called `PomodoroShortcuts`, this way we can split this logic, make it easier to test, and also if in the future we want to allow the user to enable/disable the shortcuts, it's easier to make this verification in a separate component.